src/ru/nsu/ccfit/zuev/osu/game/GameHelper.java

```diff
@@ -46,6 +46,8 @@ public class GameHelper {
     private static boolean useReplay;
     private static boolean isKiai = false;
     private static boolean auto = false;
+    private static boolean removeNoteLock = false;
+    private static boolean removeNoteLockWhenMiss = false;
```

Comment:
I'm not sure about having this setting in both `Config` and `GameHelper`. Would prefer to have it in one class instead (preferably `Config` since that's where we store settings-related values).

Additionally, these don't seem to be referenced anywhere, either.

Reply:
I forgot to delete it, thank you!